src/Application.php

```diff
@@ -131,21 +139,46 @@ public function checkConfigIsValid(array $transformationConfig): void
         $transformationValidator->validate();
     }
 
+    private function writeConfigMetadata(Config $config, Configuration $componentConfiguration, string $folder): void
+    {
+        $storageApiClient = $this->storageApiClient;
+        if ($storageApiClient instanceof Client) {
```

Comment:
Nebolo by jednoduchsie vzdy pouzivat `BranchAwareClient`, aj `main` branch ma svoje ID, a da sa pouzit rovnako ako dev-branch. Bolo by tak mensie riziko nejakych edge-cases.

Reply:
určitě bylo, ale ne vždy to funguje stejně... třeba načítání informací o tabulce ze storage

Reply:
ok